ci-operator/config/openshift/cucushift/openshift-cucushift-master.yaml

```diff
@@ -0,0 +1,20 @@
+build_root:
+  image_stream_tag:
+    name: verification-tests
```

Comment:
Is this name expected ?

Reply:
This is the build root present in this namespace, I have no idea which else to use

Reply:
ack